src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/validator.html.twig

```diff
@@ -0,0 +1,98 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% if collector.data.nb_violations > 0 or collector.data.calls|length %}
+        {% set status_color = collector.data.nb_violations ? 'red' : '' %}
+        {% set icon %}
+            {{ include('@WebProfiler/Icon/validator.svg') }}
+            <span class="sf-toolbar-value">
+                {{ collector.data.nb_violations ?: collector.data.calls|length }}
+            </span>
+        {% endset %}
+
+        {% set text %}
+            <div class="sf-toolbar-info-piece">
+                <b>Validator calls</b>
+                <span class="sf-toolbar-status">{{ collector.data.calls|length }}</span>
+            </div>
+            <div class="sf-toolbar-info-piece">
+                <b>Number of violations</b>
+                <span class="sf-toolbar-status sf-toolbar-status-{{ collector.data.nb_violations > 0 ? 'red' }}">{{ collector.data.nb_violations }}</span>
```

Comment:
`{{ collector.data.nb_violations > 0 ? 'sf-toolbar-status-red' }}` (otherwise there will be a `sf-toolbar-status-` class)